documentation/asciidoc/computers/configuration/uart.adoc

```diff
@@ -118,7 +148,14 @@ Linux devices on Raspberry Pi OS:
 | secondary UART
 |===
 
-NOTE: `/dev/serial0` and `/dev/serial1` are symbolic links which point to either `/dev/ttyS0` or `/dev/ttyAMA0`.
+`/dev/serial0` and `/dev/serial1` are symbolic links which point to either `/dev/ttyS0` or `/dev/ttyAMA0`.
```

Comment:
Again I don't think I've ever seen us use the term "mini UART"

Reply:
It is a thing - the cut-down 8250 clone - and I'm sure some of my early documentation referred to it.

Reply:
The 2711 datasheet refers to it as a mini UART. That's the terminology that we have always used.

Reply:
Fascinating. I've never heard anyone refer to it like that. But then I haven't seen the 2711 datasheet.

Reply:
See page 10 of https://datasheets.raspberrypi.com/bcm2711/bcm2711-peripherals.pdf

Reply:
The Pi 5 doesn't possess a mini UART, as I've already commented.

Reply:
I think Alasdair might be confusing what Phil and James are calling the "mini UART" (the cut-down 8250 clone in the SoCs for Pi0-4) which is also mentioned in https://www.raspberrypi.com/documentation/computers/configuration.html#configuring-uarts , with the[ 3-pin debug UART connector](https://datasheets.raspberrypi.com/debug/debug-connector-specification.pdf) on the Pi 5? (which I agree shouldn't be called a "mini UART").

Reply:
Ah! I am indeed doing that.